### PR TITLE
Enable Basic Netlify hosting and Netlify CMS

### PR DESCRIPTION
**Pull request by ethanzitting**
Fix a TypeScript error in Particles.tsx.
Add minimum configuration files necessary to enable Netlify hosting and CMS.
Add Netlify related content to README.md.
Add links to tools on README.md.

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+backend:
+    name: git-gateway
+    repo: ethanzitting/opensgf.org
```
Comment (fredlawl) [inline public/admin/config.yml:3]: Shouldn't this be open-sgf/opensgf.org?

File: public/admin/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        label: "Pages"
+        files:
+            -   label: "Home"
+                name: "home"
+                file: "content/home.md"
+                fields:
+                    - { label: "Title", name: "title", widget: "string" }
+                    - { label: "Publish Date", name: "date", widget: "datetime" }
+                    - { label: "Body", name: "body", widget: "markdown" }
+                    -   label: 'Cats'
```
Comment (fredlawl) [inline public/admin/config.yml:18]: Cats?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,17 @@
+[![Netlify Status](https://api.netlify.com/api/v1/badges/3701b349-13d2-40b8-a955-16e595d73fd1/deploy-status)](https://app.netlify.com/sites/ethanzitting-opensgf/deploys)
+
+## Dev Branch Preview
+- [Netlify](https://ethanzitting-opensgf.netlify.app/)
```
Comment (fredlawl) [inline README.md:4]: Don't really think this is necessary to include in the README

File: src/components/Particles/Particles.tsx [hunk 4]
```diff
@@ -139,5 +139,11 @@ export function ParticlesComp(): JSX.Element {
...
         loadBubblesPreset(engine);
     }, []);
-    return <Particles className={styles.particles} init={particlesInit} options={options} />;
+    return (
+        <Particles
+            className={styles.particles}
+            init={particlesInit}
+            options={options as RecursivePartial<typeof options>}
+        />
+    );
```
Comment (fredlawl) [inline src/components/Particles/Particles.tsx:148]: I don't think we should sneak this into a PR for a netlify backend.

**Review by fredlawl — CHANGES_REQUESTED** [review]
14 commits for 5 file changes? A lot of these are: Merge branch 'Open-SGF:dev. Can you rebase only the necessary commits ontop of dev?

File: public/admin/config.yml (hunk 1 above)
Comment (ethanzitting) [reply public/admin/config.yml]: Good catch, implemented!

File: public/admin/config.yml (hunk 2 above)
Comment (ethanzitting) [reply public/admin/config.yml]: Was some placeholder configuration, has been replaced with some proper configuration.

File: README.md (hunk 3 above)
Comment (ethanzitting) [reply README.md]: Sounds good, implemented!

File: src/components/Particles/Particles.tsx (hunk 4 above)
Comment (ethanzitting) [reply src/components/Particles/Particles.tsx]: Good point, I will put this in own PR, thanks!

**Comment by ethanzitting** [comment]
> 14 commits for 5 file changes? A lot of these are: Merge branch 'Open-SGF:dev. Can you rebase only the necessary commits ontop of dev?

Good point, thank you for the good feedback! I will clean this up.

**Pull request closed without merging**